history.rst

```diff
@@ -5,6 +5,24 @@
 OMERO version history
 =====================
 
+5.6.2 (July 2020)
+-----------------
+
+This release adds installation documentation for server and web on
+CentOS 8 and Ubuntu 20.04, and it removes installation instructions
+for Ubuntu 16.04.
+
+This version has been tested with:
```

Comment:
Unrelated to this PR as i realize it is the wording of all 5.6.x releases. While it makes sense for omero-py and omero-web which are managed separately, these versions of the Java components are shipped with the new bundles.

Maybe we want separate paragraphs in the future.

Reply:
I could imagine linking to all the versions (e.g. even omero-model) especially if they were links to the changelogs, though we'd likely want that to be auto-generated.